docs/development/creating-editing-renovate-presets.md

```diff
@@ -0,0 +1,38 @@
+# Creating/editing Renovate presets
+
+Renovate comes with default presets that you can find in the `lib/config/presets/internal` directory.
+You can suggest changes to the presets with a pull request.
+
+Follow the rules below to increase the chance that your pull request gets merged.
+
+## General rules
+
+1. Only create/edit presets for problems which can not be fixed upstream
+1. The internal preset must be helpful for most Renovate users
+
+### Specific rules
+
+#### Group presets
+
+File location: `lib/config/presets/internal/group.ts`
```

Comment:
Do we need to mention the precise path to the preset?
This can get outdated quick if we refactor our code later on, so maybe it's better to leave it out?